betka.py

```diff
@@ -197,11 +197,16 @@ def convert_branch_to_cwt_tool(self) -> Any:
         fields = self.downstream_branch.split("-")
         if self.os_env == "rhel7":
             self.cwt_config = f"rhel7.yaml:{fields[0]}{fields[1].replace('.','')}0"
-        else:
+        elif self.os_env == "rhel8":
             release_fields = fields[1].split(".")
             self.cwt_config = (
                 f"rhel8.yaml:{fields[0]}{release_fields[0]}.{release_fields[1]}"
             )
+        else:
```

Comment:
Just a nitpick, but I would actually use `elif` here as well to explicitly check whether rhel9 was used rather than expecting no other choice, as well as using the `else` branch to fail in case something other than rhel{7,8,9} was passed as a target